controllers/profiles/getSelfAnonymousFeedsInProfile.js

```diff
@@ -16,7 +15,11 @@ module.exports = async (req, res) => {
     if (!anonymousUserId) return ErrorResponse.e404(res, 'Anonymous user not found');
 
     const result = await Getstream.feed.getAnonymousFeeds(anonymousUserId?.user_id, limit, offset);
-    const newResult = await filterFeeds(req?.userId, result?.results || []);
+    const newResult = await filterFeeds(
+      req?.userId,
+      anonymousUserId?.user_id,
+      result?.results || []
+    );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `filterFeeds` function now takes an additional parameter `anonymousUserId?.user_id`. This is a good change as it allows the function to determine if a post belongs to an anonymous user associated with the current user. However, please ensure that all calls to `filterFeeds` across the codebase are updated to include this new parameter to maintain consistency and avoid potential errors.
```diff
-    const newResult = await filterFeeds(req?.userId, result?.results || []);
+    const newResult = await filterFeeds(
+      req?.userId,
+      anonymousUserId?.user_id,
+      result?.results || []
+    );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->